src/WebJobs.Script/Diagnostics/CategoryTraceWriterFactory.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Diagnostics;
+using System.IO;
+using Microsoft.Azure.WebJobs.Host;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    internal class CategoryTraceWriterFactory
+    {
+        private readonly string _category;
+        private readonly ScriptHostConfiguration _scriptHostConfig;
+
+        public CategoryTraceWriterFactory(string category, ScriptHostConfiguration scriptHostConfig)
```

Comment:
Can we just have these logs go to the Host logs prefixed with 'node: ' (or something) instead of this new file? It seems like we're introducing a new 'log-file-per-category' concept but for only one category. Ultimately, they'll be more useful going to App Insights anyway.